test/specs/elements/Header/HeaderSubheader-test.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import HeaderSubheader from 'src/elements/Header/HeaderSubheader'
+import * as common from 'test/specs/commonTests'
+
+describe('HeaderSubheader', () => {
+  common.rendersChildren(HeaderSubheader)
+  const wrapper = mount(<HeaderSubheader />)
+
+  it('renders a <div /> element', () => {
+    wrapper.should.have.tagName('div')
+  })
+
+  it('adds the sd-header-subheader class', () => {
+    wrapper.should.have.className('sd-header-subheader')
+  })
```

Comment:
Tests for the `sd-*` classes are also handled in the `common.isConformant()` test.  Let's add the conformance test and drop this one.

Note, with the merge into the official SUI org it seems most sane to drop these sd (stardust) classes.  There are some dependencies on them so I'm not yet sure when we'll do that, but it should happen.